README.md

```diff
@@ -1,2 +1,11 @@
 # Simple Sentiment Analysis
 This is a simple Sentiment analysis python script file for the purpose of learning.
+
+## Steps to run this script:
+
+- Save the file with sentiment.py
+- run the command python3 ./sentiment.py
+
+## Output
+
+The output will be {'neg': xxx, 'neu':yyy, 'pos': zzz, 'compound': xyz}
```

Comment:
Can you add some more details about the output?

Reply:
Sure!